spartan/aztec-network/templates/boot-node.yaml

```diff
@@ -123,11 +123,20 @@ spec:
           resources:
             {{- toYaml .Values.bootNode.resources | nindent 12 }}
       volumes:
-        - name: shared-volume
-          emptyDir: {}
         - name: scripts
           configMap:
             name: {{ include "aztec-network.fullname" . }}-deploy-contracts-script
+  volumeClaimTemplates:
+  - metadata:
+      name: shared-volume
+      labels:
+        {{- include "aztec-network.labels" . | nindent 8 }}
+    spec:
+      accessModes: ["ReadWriteOnce"]
+      resources:
+        requests:
+          storage: {{ .Values.bootNode.storage }}
+      storageClassName: "gp2"
```

Comment:
think gp3 storage is both cheaper and better throughput unless I missed something on a cursory read of aws docs

Reply:
Yes, this note is accurate. Currently `gp2` is set as the default `storageClass` deployed with a new EKS Cluster. I have removed the explicit storage class assignment, which causes the deployment to use whatever is set as default. This would let us add a `gp3` `storageClass` to the cluster and set it as default in the future. Deleting the reference also keeps compatibility with `kind` installations of the helm chart, since `gp2` classes wouldn't exist on the local machine.